### ISSUE TITLE
Screen reader does not announce if an update is available for an installed package unless you hover over the update button

### ISSUE THREAD

**Issue by eclectir** [Microsoft/PTVS#2231]
This more likely to be a usability bug than a MAS bug, but we should consider fixing it since we will be in the same part of the code while fixing other accessibility bugs.

**Comment by huguesv** [comment]
Implemented in https://github.com/Microsoft/PTVS/pull/2630
